### PR TITLE
GDB-8369 Cluster View broken toast error for lack of user permissions

### PR DESCRIPTION
**Pull request by Emskiqq**
GDB-8369 Ascertaining whether the error is a 403 and calls getError(error.data, error.status) if it is

### CODE REVIEW

**Comment by Emskiqq** [comment]
Jenkins retry build

File: src/js/angular/clustermanagement/controllers.js [hunk 1]
```diff
@@ -241,7 +241,7 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $uibMod
                 })
                 .catch((error) => {
                     const failMessage = $translate.instant('cluster_management.delete_cluster_dialog.notifications.fail_delete');
-                    const failedNodesList = Object.keys(error.data)
+                    const failedNodesList = error.status === 403 ? getError(error.data, error.status) : Object.keys(error.data)
```
Comment (yordanalexandrov) [inline src/js/angular/clustermanagement/controllers.js:244]: Is it only 403 that needs to be handled like this?

IMO all errors should be handled with `getError()`, except the one that returns the failed delete from the cluster.

Comment (yordanalexandrov) [inline src/js/angular/clustermanagement/controllers.js:244]: Also change the name of the variable if it might contain different messages

Comment (yordanalexandrov) [inline src/js/angular/clustermanagement/controllers.js:244]: The other actions in this view have this case handled. You can the same aproach here as well, so it is consistent.

**Comment by yordanalexandrov** [comment]
Also please change the name of the MR as it has nothing to do with what the changes are.

File: src/js/angular/clustermanagement/controllers.js (hunk 1 above)
Comment (yordanalexandrov) [inline src/js/angular/clustermanagement/controllers.js:244]: The other actions in this view have this case handled. You can the same aproach here as well, so it is consistent.

**Review by yordanalexandrov — CHANGES_REQUESTED** [review]

File: src/js/angular/clustermanagement/controllers.js [hunk 2]
```diff
@@ -241,9 +241,7 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $uibMod
                 })
                 .catch((error) => {
                     const failMessage = $translate.instant('cluster_management.delete_cluster_dialog.notifications.fail_delete');
-                    const failedNodesList = error.status === 403 ? getError(error.data, error.status) : Object.keys(error.data)
-                        .reduce((message, key) => message += `<div>${key} - ${error.data[key]}</div>`, '');
-                    toastr.error(failedNodesList, failMessage, {allowHtml: true});
+                    handleAddRemoveErrors(error.data, error.status, failMessage);
```
Comment (yordanalexandrov) [inline src/js/angular/clustermanagement/controllers.js:244]: Rename the function to something more generic `handleErrors` perhaps

**Review by yordanalexandrov — CHANGES_REQUESTED** [review]

**Review by yordanalexandrov — APPROVED** [review]

**Pull request merged**